examples/tti_example.py

```diff
@@ -103,22 +103,25 @@ def damp_boundary(damp, h, nbpml):
             damp[:, :, -(i + 1)] += val
 
 nrec, _ = data.traces.shape
-m = DenseData("m", model.vp.shape, dtype)
+m = DenseData(name="m", shape=model.vp.shape, dtype=dtype)
```

Comment:
Is there some python magic that we can do to eliminate the need for the name parameter and automatically pick up the name of the variable that points to this object? There are some libraries that seem to do this.